### PR TITLE
python-INGInious: add at 0.3.dev1

### PR DESCRIPTION
**Pull request by layus**
INGInious is an intelligent grader that allows secured and automated testing of code made by students.

### CODE REVIEW

**Comment by layus** [comment]
Travis fails due to the log exceeding 4MB...

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -3871,6 +3909,29 @@ let
...
+      pytestflakes pytestpep8 sphinx mock
+    ];
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/P/PyLTI/${name}.tar.gz";
+      sha256 = "1lkk6qx8yfx1h0rhi4abnd44x0wakggi6zs0nvi572lajf6ydmdh";
+    };
+
+    meta = {
+      description = "PyLTI provides Python Implementation of IMS LTI interface that works with edX";
```
Comment (jagajaga) [inline pkgs/top-level/python-packages.nix]: Please omit pkg's name in the description.

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -12742,6 +12894,20 @@ let
...
 
+  semantic-version = buildPythonPackage rec {
+    name = "semantic_version-2.4.2";
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/s/semantic_version/${name}.tar.gz";
+      md5 = "fd7d5ade76e78d8540b9a4044496a57c";
+    };
+
+    meta = {
+      description = "A library implementing the 'SemVer' scheme.";
```
Comment (jagajaga) [inline pkgs/top-level/python-packages.nix]: Please remove dot at the end.

File: pkgs/top-level/python-packages.nix [hunk 3]
```diff
@@ -3173,6 +3196,21 @@ let
...
+    name = "EasyProcess-0.1.9";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/E/EasyProcess/${name}.tar.gz";
+      md5 = "3da72e2fe16781fe5c7b3b0c6c40ee7b";
+    };
+
+    meta = {
+      description = "Easy to use python subprocess interface";
+      homepage = "https://github.com/ponty/EasyProcess";
```
Comment (pSub) [inline pkgs/top-level/python-packages.nix]: You don't need quotes around urls.

Comment (layus) [inline pkgs/top-level/python-packages.nix]: As a matter of fact, nearly all the urls are quoted in this file. The only unquoted ones do not contain antiquotations. Should I really remove these ?

**Pull request merged**